### PR TITLE
Test invest all

### PR DESCRIPTION
**Pull request by jepidoptera**
Expanded the test a bit for this strategy, because I expanded the strategy a bit.

### CODE REVIEW

File: hydradx/tests/test_omnipool_amm.py [hunk 1]
```diff
@@ -1974,3 +1974,283 @@ def test_dynamic_fees_with_trade(liquidity: list[float], lrna: list[float], orac
         expected_asset_fee = min(max(prev_asset_fees[tkn] + df, asset_fee_params['minimum']), asset_fee_params['fee_max'])
         if omnipool.last_fee[tkn] != pytest.approx(expected_asset_fee, rel=1e-15):
             raise AssertionError('Asset fee is not correct.')
+
+
+@given(asset_quantity_strategy, omnipool_config())
+def test_LP_delta_r(lp_amount, omnipool: oamm.OmnipoolState):
```
Comment (poliwop) [inline hydradx/tests/test_omnipool_amm.py:1980]: How is this related to invest all?

**Review by poliwop — COMMENTED** [review]
There appear to be many changes unrelated to the invest_all strategy in this PR, it would be good to have some description of their goal/intent or perhaps better to split them into a different PR.

File: hydradx/tests/test_omnipool_amm.py (hunk 1 above)
Comment (jepidoptera) [reply hydradx/tests/test_omnipool_amm.py:1980]: Nothing. I believe these tests are missing from the main branch, but were present in the branch I cloned this branch from. They should be added back, but maybe as part of a different pull request.
